File: src/main/java/MachTeacher/MachTeacher/dto/ProfileDto.java
package MachTeacher.MachTeacher.dto;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ProfileDto {

    private Long id;
    private String name;
    private String email;
    private String phone;
    private String location;
    private String role;
    private String bio;
    private String university;
    private String career;
    private String semester;
    private String subjects;
    private String hourlyRate;
    private String teachingExperience;
    private String aboutMe;
    private String references;
    private String avatarUrl;
}

File: MachTeacher/app/src/main/java/com/example/machteacher/dto/ProfileDto.kt
package com.example.machteacher.dto


data class ProfileDto(
    val id: Long?,
    val name: String? = null,
    val email: String? = null,
    val phone: String? = null,
    val location: String? = null,
    val role: String? = null,

    val bio: String? = null,
    val university: String? = null,
    val career: String? = null,
    val semester: String? = null,

    // Mentor extras
    val subjects: String? = null,
    val hourlyRate: String? = null,
    val teachingExperience: String? = null,
    val aboutMe: String? = null,
    val references: String? = null
)
